eng/Directory.Build.Common.props

```diff
@@ -243,6 +243,14 @@
     <RunSettingsFilePath Condition="'$(AZURE_SKIP_DEFAULT_RUN_SETTINGS)' != 'true'">$(RepoEngPath)\nunit.runsettings</RunSettingsFilePath>
   </PropertyGroup>
 
+  <!-- Set global test timeout of 5 seconds -->
+  <ItemGroup Condition="'$(IsTestProject)' == 'true' and '$(IsClientLibrary)' == 'true'">
```

Comment:
5 seconds is really short, won't this break a ton of live tests?

Reply:
We'll need an opt-out for this in Event Hubs and Service Bus, at least.

Reply:
see above comment 😄